<launch>
  <!--first-->
  <node ns = "first" pkg="icart_move_plan" type="icart_move_first" name="icart_move_first" respawn="true">
    <remap from="/ypspur_ros_first/odom" to="/icart_first/diff_drive_controller/odom"/>
    <remap from="/ypspur_ros_first/cmd_vel" to="/icart_first/diff_drive_controller/cmd_vel"/>
  </node>

  <!--second-->
  <node ns = "second" pkg="icart_move_plan" type="icart_move_second" name="icart_move_second" respawn="true">
    <remap from="/ypspur_ros_second/odom" to="/icart_second/diff_drive_controller/odom"/>
    <remap from="/ypspur_ros_second/cmd_vel" to="/icart_second/diff_drive_controller/cmd_vel"/>
  </node>
  <!--move_node-->
  <node pkg="icart_move_plan" type="path_planning_speed_control" name="path_planning">
    <remap from="/ypspur_ros_first/odom" to="/icart_first/diff_drive_controller/odom"/>
    <remap from="/ypspur_ros_second/odom" to="/icart_second/diff_drive_controller/odom"/>
  </node>

  <!--data_catch-->
  <node pkg="icart_move_plan" type="data_create" name="data_create">
    <remap from="/ypspur_ros_first/odom" to="/icart_first/diff_drive_controller/odom"/>
    <remap from="/ypspur_ros_second/odom" to="/icart_second/diff_drive_controller/odom"/>
  </node>

</launch>
